skimage/measure/tests/test_fit.py

```diff
@@ -1,9 +1,8 @@
 import numpy as np
 from numpy.testing import assert_equal, assert_almost_equal, assert_array_less
 from numpy.testing import assert_array_less
-from skimage.measure import LineModel, LineModelND, CircleModel, EllipseModel, ransac
 import pytest
-from skimage.measure import LineModelND, CircleModel, EllipseModel, ransac
+from skimage.measure import (LineModelND, CircleModel, EllipseModel, ransac)
```

Comment:
Nevertheless, I think they should go. =) It is not a common syntax convention except with multi-line imports.